command_line/reindex.py

```diff
@@ -151,8 +150,9 @@ def run(args):
 
     params, options = parser.parse_args(show_diff_phil=True)
 
-    reflections = flatten_reflections(params.input.reflections)
-    experiments = flatten_experiments(params.input.experiments)
+    reflections, experiments = reflections_and_experiments_from_files(
+        params.input.reflections, params.input.experiments
```

Comment:
Wondering about this pattern.
Is there any benefit in exposing params.input.reflections`/`params.input.experiments` to the application at all, or could/should this be integrated into `params.input`/the `parse_args` call?

Reply:
This is a good suggestion but more wide-ranging - possibly a follow up pull request?